graph/xml_parser.py

```diff
@@ -61,6 +61,19 @@ def get_used_rules(self):
                         result=result.text))
         return rules
 
+    def get_notselected_rules(self):
+        ns = {
+            'ns0': 'http://checklists.nist.gov/xccdf/1.2',
```

Comment:
I would use `xccdf:` instead of `ns0:`.

I also suggest defining the namespace dictionary on module level and move there all the namespace definitions from the whole project, I think there is a high chance that you will want to reuse them. That can be done in a separate PR, though.